assets/app/view/game/note.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module View
+  module Game
+    class Note < Snabberb::Component
+      include Actionable
+      needs :game
+
+      def render
+        round = @game.round
+        step = round.active_step
+        entity = step.current_entity
+        return '' unless entity
+
+        children = []
+        if step.respond_to?(:show_note?) && step.show_note?(entity) && step.respond_to?(:note_text_block)
```

Comment:
I tested it and we don't need the other respond_to? either.  I removed both. 

Side Note:  I also wonder if it would be useful to add the following to the `Step::Base` class, which would serve as a indication that the feature is available if others want to use it? 
```
def show_note?(_entity)
  false
end
def note_text_block
  []
end
```